File: go-tutorial/stage2/main/hello-world.go
package main

import (
	"fmt"
	"os"
	"time"
)

func printLocalTime() {
	fmt.Println(time.Now().Format(time.ANSIC))
}

func main() {
	who := "world"
	if (len(os.Args) > 1) {
		who = os.Args[1];
	}
	fmt.Println(getGreet(who))
	printLocalTime()
}

File: python-tutorial/stage3/main/hello_world.py
import sys

from lib.time import print_localtime
from main.hello_greet import get_greet


def main() -> None:
    who = "world"

    if len(sys.argv) > 1:
        who = sys.argv[1]

    print(get_greet(who))
    print_localtime()


if __name__ == "__main__":
    main()
